app/services/feature-flags.js

```diff
@@ -14,6 +14,10 @@ export default class FeatureFlagsService extends Service {
     return this.currentUser && (this.currentUser.isStaff || this.currentUser.isConceptAuthor);
   }
 
+  get canSeeLanguageGuidesForStage2() {
+    return this.getFeatureFlagValue('can-see-language-guides-for-stage-2') === 'test';
```

Comment:
@andy1li in the past, we've tended to keep the `user.isStaff` conditino in here too - that way each caller doesn't have to worry about it (we turn all feature flags on for staff for testing anyway). 
```suggestion
    return this.getFeatureFlagValue('can-see-language-guides-for-stage-2') === 'test' || this.authenticator.currentUser?.isStaff;
```